sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,11 +1,26 @@
 # Release History
 
-## 4.1.0-preview.2 (Unreleased)
+## 4.1.0 (2020-08-11)
+
+- Added the optional `serviceVersion` property to the `CertificateClient` optional parameters.
+    - It defaults to the latest supported API version, which currently is `7.1`.
+    - Other supported service version at the moment is `7.0`.
+    - If you were using `4.1.0-preview.1` version of this package, then this was previously named `apiVersion`.
+- `CertificateKeyType` now supports the `oct` type.
+    - Since the service now supports `oct` keys, now the client will correctly report the type of these keys.  
+- Added `recoverableDays` as an optional property to `CertificateProperties`.
+    - The Key Vault Keys client `4.1.0` now allows for Keys to be created with a determined number of days in which they can be recovered after deletion.
+      For this reason, the Certificates client will now show this `recoverableDays` property when reading the key in use. This is only relevant for Azure Key Vaults with the soft-delete setting enabled.
```

Comment:
We are talking about Keys client here, it should be certificates.
We don't necessarily need to call out `4.1.0` here in the sub bullet point, as that is what the whole section is about.
I would recommend the below

```suggestion
    - Use this when creating a certificate to provide the number of days in which a certificate can be recovered after deletion. The same property will reflect the set value when reading the certificates.
    - This is applicable only for Azure Key Vaults with the soft-delete setting enabled.
```

Reply:
I used this instead:

```
    - When creating and updating a certificate, use the `recoverableDays` property to set the number of days in which the certificate can be recovered after deletion. When certificates are read from the service, they will have the `recoverableDays` property in their `properties` field.
    - This is applicable only for Azure Key Vaults with the soft-delete setting enabled.
```

thoughts?